### ISSUE TITLE
ImportError: No module named request

### ISSUE THREAD

**Issue by santsys** [rapid7/nexpose-client-python#40]
When attempting to run the v0.1.5 code using Python 2.7 the following error is returned.

```text
Traceback (most recent call last):
  File ".\aruba_nexpose.py", line 15, in <module>
    from nexpose import nexpose
  File "W:\Python27\lib\site-packages\nexpose\nexpose.py", line 5, in <module>
    import urllib.request
ImportError: No module named request
```

I believe this breaks the backwards compatibility for python 2.7 due to the changes in the urllib modules between the two versions.

----

```text
> python27 --version
Python 2.7.13
```

**Comment by gschneider-r7** [comment]
Hm I thought this was working for me before I published it... 😞

**Comment by gschneider-r7** [comment]
Should be fixed in 0.1.6
